src/data/faq.json

```diff
@@ -94,8 +94,8 @@
                             "The <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b> security certificate may be disabled on your device. This is a known cause to trigger the notification:",
                             "<b>'CAUSE: 2001 Something went wrong. An error occurred while trying to establish a secure connection to the server.'</b>",
                             "Follow these steps in order to activate the security certificate (The exact path depends on the device):",

```

Comment:
The punctuation is a bit strange:
"Follow these steps in order to activate the security certificate (The exact path depends on the device):",

Perhaps:
"Follow these steps in order to activate the security certificate:",
"(The exact path depends on the device.)",  ?

Reply:
fixed by removing the paranthesis. it's just two sentences now.